### PR TITLE
Symfony 4.4 compatibility

### PR DESCRIPTION
**Pull request by rafaelpires815**
https://jira.wiley.com/browse/CKLS-12324

This pull request has all required changes to make this bundle compatible with Symfony 4.4

### CODE REVIEW

File: DataTable/Renderer/TwigRenderer.php [hunk 1]
```diff
@@ -3,12 +3,13 @@
 namespace CrossKnowledge\DataTableBundle\DataTable\Renderer;
 
 use CrossKnowledge\DataTableBundle\DataTable\Table\AbstractTable;
+use Twig\Environment;
 
 class TwigRenderer implements RendererInterface
 {
     protected $twig;
```
Comment (MatheusNaldi) [inline DataTable/Renderer/TwigRenderer.php]:
```suggestion
    /** @var Environment */
    protected $twig;
```

File: DataTable/Renderer/TwigRenderer.php [hunk 2]
```diff
@@ -3,12 +3,13 @@
...
 
-    public function __construct(\Twig_Environment $env)
+    public function __construct(Environment $env)
```
Comment (MatheusNaldi) [inline DataTable/Renderer/TwigRenderer.php]:
```suggestion
    /**
     * TwigRenderer constructor
     *
     * @param Environment $env
     */    
    public function __construct(Environment $env)
```
>Add PHPDoc blocks for all classes, methods, and functions (though you may be asked to remove PHPDoc that do not add value);

https://symfony.com/doc/3.4/contributing/code/standards.html#documentation
Also present in 4.4 version: https://symfony.com/doc/4.4/contributing/code/standards.html#documentation

File: Tests/DataTable/Column/ColumnTest.php [hunk 3]
```diff
@@ -3,12 +3,13 @@
...
 use CrossKnowledge\DataTableBundle\DataTable\Table\Element\Column\Column;
+use PHPUnit\Framework\TestCase;
 
-class ColumnTest extends \PHPUnit_Framework_TestCase
+class ColumnTest extends TestCase
 {
     public function testUndefinedOptionThrowsException()
     {
-        $this->setExpectedException('Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException');
+        $this->expectException('Symfony\Component\OptionsResolver\Exception\UndefinedOptionsException');
```
Comment (MatheusNaldi) [inline Tests/DataTable/Column/ColumnTest.php]:
```suggestion
        $this->expectException(UndefinedOptionsException::class);
```
Is it possible if we declare this class in the uses block and use ::class here?

File: Tests/DataTable/RegistryTest.php [hunk 4]
```diff
@@ -18,7 +19,7 @@ public function testRetrieveTableById()
         $registry = new DataTableRegistry(['test' => $table]);
         $this->assertEquals($table, $registry->retrieveByTableId('test'));
 
-        $this->setExpectedException('\BadMethodCallException');
+        $this->expectException('\BadMethodCallException');
```
Comment (MatheusNaldi) [inline Tests/DataTable/RegistryTest.php]:
```suggestion
        $this->expectException(BadMethodCallException::class);
```
Is it possible if we declare this class in the uses block and use ::class here?

File: Tests/DataTable/RegistryTest.php [hunk 5]
```diff
@@ -18,7 +19,7 @@ public function testRetrieveTableById()
...
         $registry->retrieveByTableId('undef');
     }
 }
```
Comment (MatheusNaldi) [inline Tests/DataTable/RegistryTest.php]:
```suggestion
}

```
Please add a blank line before EOF

File: Tests/Twig/DataTableExtensionTest.php [hunk 6]
```diff
@@ -6,14 +6,16 @@
...
 
 require_once __DIR__.'/../UsesContainerTrait.php';
 
-class DataTableExtensionTest extends \PHPUnit_Framework_TestCase
+class DataTableExtensionTest extends TestCase
 {
     use UsesContainerTrait;
 
-    protected function getFunctionByName(\Twig_Extension $extension, $name)
+    protected function getFunctionByName(AbstractExtension $extension, $name)
```
Comment (MatheusNaldi) [inline Tests/Twig/DataTableExtensionTest.php]:
```suggestion
    /**
     * Return function of extension by name
     *
     * @param AbstratExtension $extension
     * @param string           $name
     */
    protected function getFunctionByName(AbstractExtension $extension, string $name)
```
Please add documentation

File: Tests/Twig/DataTableExtensionTest.php [hunk 7]
```diff
@@ -6,14 +6,16 @@
...
     {
         foreach ($extension->getFunctions() as $function) {
             if ($function->getName()==$name) {
```
Comment (MatheusNaldi) [inline Tests/Twig/DataTableExtensionTest.php]: If possible, could you please fix line 30 the same way you did in the Tests/DataTable/Renderer/TwigRendererTest.php file using SomeClass::class? 

(Sorry for the comment here, I can't write a comment in that line as you didn't change anything near there)

**Review by MatheusNaldi — CHANGES_REQUESTED** [review]

File: DataTable/Renderer/TwigRenderer.php (hunk 1 above)
Comment (rafaelpires815) [reply DataTable/Renderer/TwigRenderer.php]: Done.

File: DataTable/Renderer/TwigRenderer.php (hunk 2 above)
Comment (rafaelpires815) [reply DataTable/Renderer/TwigRenderer.php]: Done.

File: Tests/DataTable/Column/ColumnTest.php (hunk 3 above)
Comment (rafaelpires815) [reply Tests/DataTable/Column/ColumnTest.php]: Done.

File: Tests/DataTable/RegistryTest.php (hunk 4 above)
Comment (rafaelpires815) [reply Tests/DataTable/RegistryTest.php]: Done.

File: Tests/DataTable/RegistryTest.php (hunk 5 above)
Comment (rafaelpires815) [reply Tests/DataTable/RegistryTest.php]: Done.

File: Tests/Twig/DataTableExtensionTest.php (hunk 6 above)
Comment (rafaelpires815) [reply Tests/Twig/DataTableExtensionTest.php]: Done.

File: Tests/Twig/DataTableExtensionTest.php (hunk 7 above)
Comment (rafaelpires815) [reply Tests/Twig/DataTableExtensionTest.php]: Done.

**Review by MatheusNaldi — APPROVED** [review]
Review OK!

**Pull request merged**